src/run/cluster.cpp

```diff
@@ -89,26 +188,22 @@ vector<int> cluster(DatabaseFile &db, const vector<bool> *filter) {
 	opt.db_filter = filter;
 
 	Workflow::Search::run(opt);
-	
+
 	return Util::Algo::greedy_vortex_cover(nb);
 }
 
-void run() {
```

Comment:
Probably should use a separate .cpp file for each clustering algorithm.

Reply:
Done. This change made it necessary to implement the command-line option as well. I have done this with option number 3 given above. If you want something else, please let me know.